etc/kubernetes/helm/enterprise-gateway/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Enterprise Gateway image name and tag to use.
+image: elyra/enterprise-gateway:dev
+# Enterprise Gateway image pull policy.
+imagePullPolicy: IfNotPresent
+# The primary port on which Enterprise Gateway is servicing requests.
+port: 8888
+# Update to deploy multiple replicas of EG.
+replicas: 1
+# Log output level.
+logLevel: DEBUG
+# Whether to mirror working directories.
+mirrorWorkingDirs: false
+# Master public IP on which to expose EG.
+k8sMasterPublicIP:
+
+kernel:
+  # Kernel cluster role created by this chart.
+  clusterRole: kernel-controller
+  # Will start kernels in the same namespace as EG if True. Otherwise kernel will have its own
+  sharedNamespace: false
```

Comment:
This sounds a bit ambiguous to me since users that "bring their own namespace" could be considered shared as well.  However about something like `share_gateway_namespace`?  If so, we should probably update the use of the `shared_namespace` variable in k8s.py.